src/Api/Endpoints/Health.cs

```diff
@@ -1,29 +1,12 @@
-﻿using System.Diagnostics;
-using System.Reflection;
 using Passwordless.Service.Helpers;
-using Passwordless.Service.Storage.Ef;
 
-namespace Passwordless.Api.Endpoints;
+namespace Passwordless.Server.Endpoints;
 
 public static class HealthEndpoints
 {
     public static void MapHealthEndpoints(this WebApplication app)
```

Comment:
Maybe this method/class should be removed now that we have `HealthCheckBootstrap.MapHealthChecks`?

Reply:
Yeah I will do that separately. Maybe move to DevelopmentEndpoints. And selectively expose the endpoints depending on environment.